.env.example

```diff
@@ -329,4 +329,5 @@ APP_ROUTER_APPS_ENABLED=0
 APP_ROUTER_VIDEO_ENABLED=0
 
 # disable setry server source maps
-SENTRY_DISABLE_SERVER_WEBPACK_PLUGIN=1
\ No newline at end of file
+SENTRY_DISABLE_SERVER_WEBPACK_PLUGIN=1
+FLAGSMITH_ENVIRONMENT_ID=2KQbbhDBpJL6jxHhFYqbfa
```

Comment:
you sure this should be in here?

Reply:
> you sure this should be in here?

this would probably change in future, it is here just for the testing purpose
is it wrong? not very sure. :sweat_smile: